onnxscript/rewriter/generic_pattern_test.py

```diff
@@ -546,8 +546,8 @@ def transpose_transpose_mapping(perm0, perm1):
             # replace by return [perm0[p] for p in perm1] ?
             return new_perm
 
-        def transpose_transpose_check(op, **_) -> bool:
-            return True
+        def transpose_transpose_check(op, **_) -> pattern.MatchResult:
+            return pattern.MatchResult()
```

Comment:
I believe that returning True here should work, correct? Just a minor nit: I think "True" is more readable than "pattern.MatchResult()" and more efficient (no need to construct a new object with sub-objects). If we really want a MatchResult, we could also get the same advantages using a readonly object like `pattern.SUCCESS`